tap_stripe/__init__.py

```diff
@@ -85,6 +85,9 @@
     'transfers': {'type': 'transfer.*', 'object': ['transfer']},
     'disputes': {'type': 'charge.dispute.*', 'object': ['dispute']},
     'products': {'type': 'product.*', 'object': ['product']},
+    'invoice_line_items': {'type': 'invoice.*', 'object': ['invoice']},
+    'subscription_items': {'type': 'customer.subscription.*', 'object': ['subscription']},
+    'payout_transactions': {'type': 'payout.*', 'object': ['transfer', 'payout']},
     # Cannot find evidence of these streams having events associated:
     # subscription_items - appears on subscriptions events
```

Comment:
As we added this can we remove this comment?

Reply:
Updated the map and removed the comment